kube_hunter/modules/discovery/apiserver.py

```diff
@@ -49,7 +49,7 @@ def execute(self):
 
     def has_api_behaviour(self, protocol):
         try:
-            r = self.session.get("{}://{}:{}".format(protocol, self.event.host, self.event.port))
+            r = self.session.get("{}://{}:{}".format(protocol, self.event.host, self.event.port), timeout=30)
```

Comment:
Use a configurable value instead of hard coded one.

Reply:
Done.